indigo_app/templates/indigo_app/tasks/available_tasks.html

```diff
@@ -69,7 +69,12 @@ <h4>Available tasks <span class="badge badge-light">{{ paginator.count }}</span>
               <i class="fas fa-sm fa-fw task-icon-pending_review"></i> Pending review
             </label>
           </div>
-
+          <select class="form-control selectpicker col-2 ml-2" id="{{ form.type.id_for_label }}" name="{{ form.type.html_name }}"
```

Comment:
Perhaps make this a multiple select field (eg. a user probably doesn't care if the commencement linking is active or passive).

Will require the `multiple` keyword here and a change to the form type in Django.

Reply:
hmm. that does make it a bit less neat in terms of clearing it, but you're right. maybe the codes should be fewer and we can differentiate between passive / active ones some other way?

Reply:
but let's just get this out for the time being, we can improve on it once people use it :grin:

Reply:
See "Select/Deselect All" option here: https://developer.snapappointments.com/bootstrap-select/examples/

Reply:
If you set the `title` attribute then it will show a decent placeholder, and you don't need the blank/all option (which doesn't make sense when you can select multiple options). (see Placeholder here https://developer.snapappointments.com/bootstrap-select/examples/)

Reply:
Thanks so much! This is done now